pkgs/by-name/py/pyditz/package.nix

```diff
@@ -12,21 +12,21 @@ python3Packages.buildPythonApplication rec {
     inherit pname version;
     hash = "sha256-2gNlrpBk4wxKJ1JvsNeoAv2lyGUc2mmQ0Xvn7eiaJVE=";
   };
+
   nativeBuildInputs = with python3Packages; [ setuptools-scm ];
+
   propagatedBuildInputs = with python3Packages; [
     pyyaml
     six
     jinja2
     cerberus
   ];
 
-  nativeCheckInputs = with python3Packages; [ unittestCheckHook ];
```

Comment:
Could you please check if we can run the tests? Perhaps it's required to switch from PyPI to the upstream source.

-> https://hg.sr.ht/~zondo/pyditz/browse/tests?rev=tip